pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -610,6 +627,7 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 //  4. Create sandbox if necessary.
 //  5. Create init containers.
 //  6. Create normal containers.
+//  7. Create ephemeral containers.
```

Comment:
This seems to contradict the comment you had above " Ephemeral containers may be started even if initialization is not yet complete."

If we want to allow the ephemeral containers to be created even when init containers haven't finished running, shouldn't we create them before (5)?